routers/static_pages.py

```diff
@@ -38,8 +38,11 @@ def serve_static_file(request: Request, path: str):
 
     # if the path has no extension, try to serve a .html file
     if not os.path.splitext(gcs_path)[1]:
-        html_url = bucket.blob(gcs_path + ".html").public_url
-        r = requests.get(html_url)
+        html_blob = bucket.blob(gcs_path + ".html")
+        if not html_blob.exists():
```

Comment:
I don't think you wanna do this, because it means a static file that doesn't have an extension but is NOT an HTML file cannot be served at all